service/src/test/java/uk/nhs/adaptors/gp2gp/ehr/mapper/PertinentInformationObservationValueMapperTest.java

```diff
@@ -44,8 +44,8 @@
     private static final String EXPECTED_RANGE = "Range Value: Low 10.1 test-unit High 20.5 test-unit ";
     private static final String EXPECTED_RATIO = "Ratio Value: < 10 test-unit / > 20 test-unit ";
     private static final String EXPECTED_TIME = "Time Value: 12:30:10 ";
-    private static final String EXPECTED_DATE_TIME = "DateTime Value: Wed Mar 01 12:50:12 GMT 2017 ";
-    private static final String EXPECTED_PERIOD = "Period Value: Start Wed Mar 01 12:50:12 GMT 2017 End Sun Jun 03 13:12:10 GMT 2018 ";
+    private static final String EXPECTED_DATE_TIME = "DateTime Value: 2017-03-01 12:50:12 ";
+    private static final String EXPECTED_PERIOD = "Period Value: Start 2017-03-01 12:50:12 End 2018-06-03 14:12:10 ";
```

Comment:
why time changed without changing input data?

Reply:
It casted to UK time now.